src/parser/scanner.lex

```diff
@@ -471,17 +473,6 @@ CHINESE_LABEL               ({U2}{U}|{U3}{U}{U}|{U4}{U}{U}{U})+
                                 // Must match /* */
                                 throw GraphParser::syntax_error(*yylloc, "unterminated comment");
                             }
-\`{CHINESE_LABEL}\`         {
```

Comment:
Where is the guarantee that Chinese characters must be surrounded by backquote?

Reply:
Both neo4j and mysql can use Chinese characters directly, do we have to use backquote?

Reply:
Why Chinese characters must be surrounded by backquote?

Reply:
OK. no problem. I checked the two methods are also supported in mysql.